doc/policies/conditions.rst

```diff
@@ -115,6 +115,21 @@ The tokeninfo condition works the same way as userinfo but matches the tokeninfo
 .. note:: Similar to the userinfo condition, a policy with an active tokeninfo condition will
    throw an exception whenever the token object cannot be determined (usually from the serial).
 
+``token``
+^^^^^^^^^
+
+The token condition works on the database columns of the token. This would be
+``description``, ``otplen``, ``count``, ``serial`` but most importantly
+also ``failcount`` and ``tokentype``.
+
+.. note:: A policy with an active tokeninfo condition will
+   throw an exception whenever the token object cannot be determined.
+   It will also throw an error, if the request ``Key`` does not exist
+   as a database column.
+
+.. note:: The matching is case sensitive. Note, that e.g. token types are
+   stored in lower case in the database.
+
```

Comment:
Maybe add a note to not add a token condition to admin policies.

Reply:
This is currently also true for other conditions (lile ``userinfo``).
I am planing to solve this in #2676 for 3.6. So I will not add a warning in the docs for 3.6, which will hopefully not be necessary when releasing 3.6 anymore.

Reply:
okay